ollama/_client.py

```diff
@@ -725,5 +733,9 @@ def _parse_host(host: Optional[str]) -> str:
   split = urllib.parse.urlsplit('://'.join([scheme, hostport]))
   host = split.hostname or '127.0.0.1'
   port = split.port or port
+  path = split.path or ''
```

Comment:
This is unnecessary now since `path` is assigned in the if-block below

Reply:
oops. fixed!